torch/csrc/jit/passes/tensorexpr_fuser.cpp

```diff
@@ -1258,6 +1258,10 @@ Operation createTensorExprOp(const Node* node) {
   if (node->hasAttribute(attr::symbolic_shape_inputs)) {
     sym_shapes = node->is(attr::symbolic_shape_inputs);
   }
+  bool allow_stack_outputs = false;
```

Comment:
This bool should be used below before calling `runWithAllocatedOutputs`.

Reply:
thanks yea - I didn't mean to push this commit out yet... still have to add test